docs/installing-open-mpi/configure-cli-options/mpi.rst

```diff
@@ -75,7 +75,9 @@ MPI API behaviors that can be used with ``configure``:
   component
 
 * ``--disable-io-romio``:
-  Disable the ROMIO MPI-IO component
+  Disable the :ref:`ROMIO MPI-IO <label-romio-reference>` component
+  from being compiled.This is an outdated mechanism and functions
+  similarly to ``--enable-mca-no-build`` for the romio341 component.
```

Comment:
```suggestion
  from being compiled.  This is a deprecated mechanism; it is effectively the
  same as adding ``io-romio341`` to the list of items passed to ``--enable-mca-no-build``.
```

Reply:
Does anyone else find it disturbing that the user has to know the version of ROMIO embedded in the OMPI code base just so they can disable it?

Reply:
I think the problem is more that the romio component was poorly named

Reply:
> I think the problem is more that the romio component was poorly named

I think you're right; when this component was initially created, we hadn't envisioned anything other than ROMIO, and we had envisioned being able to support more than one version of ROMIO simultaneously.

Neither of those have really turned out that way (i.e., we emphasize OMPIO these days, and I think we've only ever had multiple versions of ROMIO for developers -- not end users).  I wonder if we should take a future action item to rename this component `romio` and add an MCA component name synonym in for `romio341` (for backwards compatibility -- similar to what we did for `vader`).